pandas/tests/generic/test_series.py

```diff
@@ -227,3 +227,9 @@ def test_valid_deprecated(self):
         # GH18800
         with tm.assert_produces_warning(FutureWarning):
             pd.Series([]).valid()
+
+    def test_shift_always_copy(self):
+        # GH22397
+        s = Series([1, 2, 3])
```

Comment:
can u also test for datetime types

Reply:
Done (assuming DatetimeIndex was meant, which required another fix) if you just meant a series of datetimes, will change - let me know